examples/systemd/post-upgrade

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# this post upgrade script is run each time the teleport package is upgraded
+
+set -eu
+
+# skip reload and restart when systemd is disabled. This is only relevant when
+# testing in a container.
+if systemctl status > /dev/null 2>&1; then
+    # reload systemd configuration
+    systemctl daemon-reload
```

Comment:
Is there a way to reload just Teleport-related configs? Does that make sense?

Reply:
I'm not sure there is a way to reload just the teleport related configs. It seems like system daemon-reload is the common pattern.